src/ax/message_test.go

```diff
@@ -0,0 +1,18 @@
+package ax_test
```

Comment:
Is this file supposed to be all commented out?

Reply:
It's not supposed to exist at all!